ViewModel/TableWithPlot/Rows/GraphicalRow.cs

```diff
@@ -109,5 +109,15 @@ public void SetConfidenceLimits(double conf05, double conf95)
             NotifyPropertyChanged(nameof(Confidence95));
             NotifyPropertyChanged(nameof(Confidence05));
         }
+
+        public override void UpdateRow(int col, double value)
+        {
+            switch (col)
+            {
+                case 0:
```

Comment:
No Y?

Reply:
I think this is fine for now. I don't think the graphical row is getting used. Will, said he didn't like this approach to copy and paste and wants to scrap it. I thought it was a good idea to submit it so that we would have it in the history, but i think he is going to rip it out anyway.